examples/configuration/src/index.ts

```diff
@@ -0,0 +1,24 @@
+import { DaprClient, /* CommunicationProtocolEnum */ } from "dapr-client";
+
+// Common settings
```

Comment:
We can remove the commented sections of the code.

Reply:
@amulyavarote, I think one rationale to keep it was to allow users to uncomment if they are using GRPC. This is also a common pattern we usually get to see in examples:
```js
import { DaprClient } from "dapr-client";

// Uncomment if you want to use gRPC
// import { CommunicationProtocolEnum } from "dapr-client"; 
```

But removing it also works, keeping the example tidy is also important.

Reply:
Works!

Reply:
sounds good, updating to reflect this -- thanks guys!